docs/frontend-shared.md

```diff
@@ -1,67 +1,39 @@
 # Frontend Shared Package
 
-
 The `frontend-shared` package is a library of shared utilities, components and styles common to multiple Hypothesis front-end applications.
 
-The `frontend-shared` package source is located in `./frontend-shared` in this repository.
-
-Note: The long-term goal is to move this package into its own repository. There are some short-term advantages to the current setup: it piggy-backs on existing tooling and gives developers the benefit of working with just one repository when making changes (which primarily benefit the `client` in the first place).
-
-## How it works
-
-Frontend-shared works by creating a symlink via `$ yarn link` between the working dir
-`./frontend-shared/` and the installed dir `node_modules/@hypothesis/frontend-shared/`.
-
 ## Client setup
 
-To set up `frontend-shared` in the client, run
+To set up ``frontend-shared`` in the client, first build the package and then publish to a your local yalc repository. You may need to install yalc first. ``yarn global add yalc ``
 
 ```shell
-$ make dev
+# from the frontend-shared dir
+$ yarn build
+$ yalc publish
 ```
 
-as normal for development in the `client` application.
-
-This will trigger the `frontend-shared` build which:
-- Builds transpiled JSX into the `frontend-shared/lib` directory
-- Creates a linked package in the `node_modules/@hypothesis` folder (this is linked to `frontend-shared/lib`)
-
-When `$ make dev` is running, file changes will automatically trigger a rebuild. To rebuild manually, run:
+In the ``client`` repsitory, link to you local pushed reposotry
 
 ```shell
-$ gulp setup-frontend-shared
+# from the client dir
+$ yalc add  @hypothesis/frontend-shared
```

Comment:
Running `yalc add` modifies the `package.json` file in the client repo, which I think is going to lead someone to accidentally commit those changes at some point. I normally use `yalc link` which replaces the version of the package installed from npm in node_modules with a symlink to `.yalc/<package name>`. Can you see any problems with using `yalc link` instead?

Reply:
Yep... that was an oversite, this is very much still primordial for me :)